python-sdk/src/astro/databases/__init__.py

```diff
@@ -1,10 +1,12 @@
 import importlib
 from pathlib import Path
+from typing import TYPE_CHECKING
 
-from airflow.hooks.base import BaseHook
-from astro.databases.base import BaseDatabase
```

Comment:
These two were the main cause of slowness !

`BaseDatabase` is just used for Type Hints !